style-about.css

```diff
@@ -0,0 +1,680 @@
+  .nav2 a:hover {
+    text-decoration: none;
+    font-weight: 700;
+    color: var(--main-color-orange);
+  } */
+
+  .nav2 .container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-right: 0;
+  }
+
+  .nav2 ul li {
+    padding: 0 1vw;
+    border: none;
+    margin: 0;
+  }
+
+  .nav2 ul li:hover {
+    transform: scale(1.2);
+  }
+
+  .nav1 ul li:hover {
+    transform: scale(1.1);
+  }
+
+  .nav2 ul li a:hover {
+    color: var(--main-color-orange);
+    background-color: #fff;
+  }
+
+  .nav1 ul .fb :hover {
+    transform: scale(1.1);
+    background-color: #1877f2;
+  }
+
+  .nav1 ul .twt :hover {
+    transform: scale(1.1);
+    background-color: #1d9bf0;
+  }
+
+  .Campaign {
+    border: 4px solid var(--main-color-orange);
+    padding: 7px 3px;
+    border-radius: 5px;
+  }
+
+  .header .tab-active a {
+    color: var(--main-color-orange);
+  }
+
+  /* Hero Section */
+  .hero {
+    background-image: url(./assets/bg1.jpg);
+  }
+
+  .hero .a2 {
+    top: 40vw;
+    left: 3vw;
+  }
+
+  .hero .a3 {
+    top: 150px;
+    right: 50px;
+  }
+
+  .hero .container {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .text-main {
+    padding-top: 0;
+    margin-top: 185px;
+  }
+
+  .text-main p {
+    display: none;
+  }
+
+  .text-main h1 {
+    font-size: 4vw;
+  }
+
+  .para {
+    padding: 1vw;
+    width: 55vw;
+    margin: 15px 0 35px 0;
+  }
+
+  .para p {
+    font-size: 1vw;
+  }
+
+  .nextdate {
+    padding-bottom: 0;
+    margin-bottom: 10vw;
+  }
+
+  .nextdate p {
+    font-size: 1.05vw;
+    padding: 5px 50px;
+    font-weight: 400;
+    line-height: 1.6;
+    color: var(--main-color-black);
+  }
+
+  /* Course Mentors */
+  .mentors h2 {
+    font-size: 1.6vw;
+    padding-top: 4vw;
+  }
+
+  .mentors .line {
+    border-width: 3px;
+    width: 5vw;
+    margin-top: 0.5vw;
+    margin-bottom: 1vw;
+  }
+
+  .mentors p {
+    font-size: 1vw;
+    padding-bottom: 0;
+    margin-bottom: 0;
+  }
+
+  .loremtxt {
+    display: block;
+    margin-top: 0;
+    padding: 0 16vw !important;
+  }
+
+  .mentors img {
+    border: 1px solid rgb(237, 235, 235);
+    width: 30%;
+    margin-top: 3vw;
+    margin-bottom: 4vw;
+  }
+
+  /* past camps */
+  .camps {
+    flex-direction: row;
+    gap: 15px;
+    margin-bottom: 20px;
+    padding: 25px;
+  }
+
+  .camp-card1,
+  .camp-card2 {
+    width: 25vw;
+    height: 13vw;
+  }
+
+  /* Partners */
+  .partner {
+    padding-bottom: 4vw;
+  }
+
+  .partner h2 {
+    font-size: 2.7vw;
+    padding-top: 4vw;
+    padding-bottom: 5px;
+  }
+
+  .partner .line {
+    border-width: 3px;
+    width: 5vw;
+    margin-top: 0.5vw;
+    margin-bottom: 2vw;
+  }
+
+  .partner .container img {
+    width: 13vw;
+  }
+
+  .partner1,
+  .partner2 {
+    display: none;
+  }
+
+  .partner-all {
+    display: block;
+  }
+
+  /* License */
+  .license {
+    padding: 1vw 0;
+  }
```

Comment:
- [ ] As requested by previous reviewers, Kindly make sure your `footer` on the `About` page has a dark background image in order to match the provided Behance design.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/235314022-19f15581-9fc5-4cba-bb3c-eafbe111ea1a.png)

   - ### BEHANCE DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/235313992-f9c38be4-57e9-4d40-9a76-7a8bb1a586de.png)